src/js/mocks/server.js

```diff
@@ -0,0 +1,105 @@
+const jsonServer = require('json-server')
+const fs = require('fs');
+const server = jsonServer.create()
+const router = jsonServer.router('db.json')
```

Comment:
this creates an empty file which just lies around. Can you just pass it an empty object here instead?